src/Aspire.Hosting.SqlServer/SqlServerDatabaseResource.cs

```diff
@@ -6,15 +6,12 @@ namespace Aspire.Hosting.ApplicationModel;
 /// <summary>
 /// A resource that represents a SQL Server database that is a child of a SQL Server container resource.
 /// </summary>
-/// <param name="name">The name of the resource.</param>
-/// <param name="databaseName">The database name.</param>
-/// <param name="parent">The parent SQL Server server resource.</param>
-public class SqlServerDatabaseResource(string name, string databaseName, SqlServerServerResource parent) : Resource(name), IResourceWithParent<SqlServerServerResource>, IResourceWithConnectionString
+public class SqlServerDatabaseResource : Resource, IResourceWithParent<SqlServerServerResource>, IResourceWithConnectionString
 {
     /// <summary>
     /// Gets the parent SQL Server container resource.
     /// </summary>
-    public SqlServerServerResource Parent { get; } = parent;
+    public SqlServerServerResource Parent { get; init; }
```

Comment:
I don't think we need `init` modifiers here. Keeping the setter undeclared (private) will work and won't change the api surface.

Reply:
Doing it myself.